pyclient/README.md

```diff
@@ -0,0 +1,61 @@
+
+# Deephaven Python Client 
+
+Deephaven Python Client is a Python package by Deephaven Data Labs. It is a client API that allows Python applications to remotely access Deephaven data servers.
+
+## Source Directory
+
+### From the deephaven-core repository root
+``` shell
+$ cd pyclient
+```
+## Dev environment setup
+``` shell
+$ pip3 install -r pyclient/requirements.txt
+```
+
+## Build
+``` shell
+$ python3 setup.py
+```
+## Run tests
+``` shell
+$ python3 -m unittest discover tests
+```
+## Install
+``` shell
+$ pip3 install dist/pydeephaven-0.0.1-py3-none-any.whl
+```
+## Quick start
+
+```python    
+    >>> from pydeephaven import Session
+    >>> session = Session() # assuming Deephaven Community Edition is running locally with the default configuration
+    >>> table1 = session.time_table(period=1000000).update(column_specs=["Col1 = i % 2"])
+    >>> daf = table1.snapshot().to_pandas()
+    >>> print(df)
```

Comment:
Example does not run, do you mean `print(daf)`, or `df = table1...` ?

Reply:
sorry, typo, should be df (dataframe)